model-server-test/src/test/kotlin/IgnitePostgresRepositoryRemovalTest.kt

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2024.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.modelix.model.lazy.RepositoryId
+import org.modelix.model.server.store.IgniteStoreClient
+import org.modelix.model.server.store.ObjectInRepository
+import java.sql.DriverManager
+import kotlin.test.AfterTest
+import kotlin.test.assertEquals
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+class IgnitePostgresRepositoryRemovalTest {
+    private fun getDbConnection() = DriverManager.getConnection(System.getProperty("jdbc.url"), "modelix", "modelix")
```

Comment:
Why do we create a new connection on every access? Could be kept alive for all test cases.

Reply:
fixed